install.sh

```diff
@@ -241,22 +241,22 @@ nvm_check_global_modules() {
 
   if [ "${MODULE_COUNT}" != '0' ]; then
     cat <<-'END_MESSAGE'
-	=> You currently have modules installed globally with `npm`. These will no
-	=> longer be linked to the active version of Node when you install a new node
-	=> with `nvm`; and they may (depending on how you construct your `$PATH`)
-	=> override the binaries of modules installed with `nvm`:
+    => You currently have modules installed globally with `npm`. These will no
```

Comment:
I don't mind the mix as long as shellcheck passes, but sure, using `nvm_echo` is fine too.